### PR TITLE
draft of doc of stand-alone transfer

### PR DESCRIPTION
**Pull request by craigsteffen**
Creating stand-alone documentation for the transfer application.

### Description

Expanded the "run" documentation page, as requested in meetings, to
document the stand-alone use of the transfer application that invokes
the transfer api under the hood.

### Motivation and Context

Requested in meetings to illustrate the use as part of
debugging jobs.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Testing (addition of new tests or update to current tests)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the UnifyFS code style requirements.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] All commit messages are properly formatted.

### CODE REVIEW

File: docs/run.rst [hunk 1]
```diff
@@ -143,3 +150,21 @@ Here is an example of a valid stage-in manifest file:
 ``/scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat``
 ``/home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf``
 ``"/home/users/me/file with space.dat" "/unifyfs/file with space.dat"``
+
+-----------------------------------------------
+  Stand-alone file stage application
+-----------------------------------------------
+The manifest subsystem can also be used during the job by
+invoking stand-alone transfer executable.  The command to use is
+"unify-static".  It's used similarly to the unix
```
Comment (MichaelBrim) [inline docs/run.rst:159]: did you mean `unifyfs-transfer-static`?

Comment (MichaelBrim) [reply docs/run.rst:159]: I guess the name of the example program is actually `transfer-static`

Comment (craigsteffen) [reply docs/run.rst:159]: Thank you.  Yes.  I was looking at the file "transfer-static" on catalyst, and my fingers typed "unify-static".  :-/  Oops.  I'll fix it.

Comment (craigsteffen) [reply docs/run.rst]: Fixed.

**Comment by craigsteffen** [comment]
I believe this now has the changes we talked about recently.

File: docs/run.rst [hunk 2]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
 ``/scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat``
 ``/home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf``
 ``"/home/users/me/file with space.dat" "/unifyfs/file with space.dat"``
+
+The transfer API stage functionality can also be used with a stand-alone
+application called "unifyfs-stage".  This function takes the argument
+of the name of a manifest file.  It can be run at any time, so could
+be run during a job, to transfer new into into the UnifyFS volume,
+or trasnfer results out and checked before the job ends.
```
Comment (MichaelBrim) [inline docs/run.rst:161]: s/trasnfer/transfer/

File: docs/run.rst [hunk 3]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
 ``/scratch/users/me/input_data/input_1.dat /unifyfs/input/input_1.dat``
 ``/home/users/me/configuration/run_12345.conf /unifyfs/config/run_12345.conf``
 ``"/home/users/me/file with space.dat" "/unifyfs/file with space.dat"``
+
+The transfer API stage functionality can also be used with a stand-alone
+application called "unifyfs-stage".  This function takes the argument
+of the name of a manifest file.  It can be run at any time, so could
+be run during a job, to transfer new into into the UnifyFS volume,
```
Comment (MichaelBrim) [inline docs/run.rst:160]: "transfer new" what?

File: docs/run.rst (hunk 2 above)
Comment (MichaelBrim) [reply docs/run.rst:161]: why "and checked"?

File: docs/run.rst [hunk 4]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
...
+or trasnfer results out and checked before the job ends.
+
+  Stand-alone file stage application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The manifest subsystem can also be used during the job by
+invoking stand-alone transfer executable.  The command to use is
+"transfer-static".  It's used similarly to the unix
+"cp" command, with source and destination, except that unify-static
```
Comment (MichaelBrim) [inline docs/run.rst:168]: s/unify-static/transfer-static/

File: docs/run.rst [hunk 5]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
...
+The transfer API stage functionality can also be used with a stand-alone
+application called "unifyfs-stage".  This function takes the argument
+of the name of a manifest file.  It can be run at any time, so could
+be run during a job, to transfer new into into the UnifyFS volume,
+or trasnfer results out and checked before the job ends.
+
+  Stand-alone file stage application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The manifest subsystem can also be used during the job by
+invoking stand-alone transfer executable.  The command to use is
```
Comment (MichaelBrim) [inline docs/run.rst:166]: invoking "the" stand-alone

File: docs/run.rst [hunk 6]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
...
+"transfer-static".  It's used similarly to the unix
+"cp" command, with source and destination, except that unify-static
+is aware that it is copying files between the external parallel
+file system and the internal UnifyFS volume.
+
+The stand-alone can be invoked at any time the UnifyFS servers
```
Comment (MichaelBrim) [inline docs/run.rst:172]: The stand-alone "transfer" can be

File: docs/run.rst [hunk 7]
```diff
@@ -143,3 +153,31 @@ Here is an example of a valid stage-in manifest file:
...
+are up and responding to requests.  Thus the user can move files
+in or out during a job where UnifyFS is running at any time
+during the job, if UnifyFS has been "start"ed as above.  This
+could be done during the job to bring in new input, or perhaps
+transfer results files out so they can be verified before they
+job terminates.
+
+Examples of use of the transfer-static command:
+srun -N 2 /path/to/libexec/transfer-static /path/on/parallelfs/file.dat /unifyfs/file.dat
```
Comment (MichaelBrim) [inline docs/run.rst:181]: We may want to comment on how it is a parallel transfer, which helps to explain the '-N 2' arg to srun

**Comment by CamStan** [comment]
This work and requested fixes have been incorporated into #685

**Pull request closed without merging**